lib/plugins/aws/package/compile/events/apiGateway/lib/method/fixtures.test.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
```

Comment:
Let's maybe rename this file into `requestParameters.test.js`

(we do not use a convention of naming files that use fixtures as _fixtures.test.js_, it's more about what tests cover)

Reply:
That is certainly no trouble at all. I'm also happy to move the tests into `index.test.js` as you mentioned elsewhere if that is preferable.